src/config/chain.config.ts

```diff
@@ -35,7 +35,7 @@ const chainConfig: Record<number, ChainConfig> = {
     [VOLTA_CHAIN_ID]: {
         chainName: Chain.VOLTA,
         chainDisplayName: "Energy Web Volta Testnet",
-        rpcUrl: "https://volta-rpc.energyweb.org",
+        rpcUrl: "https://volta-rpc-vkn5r5zx4ke71f9hcu0c.energyweb.org/",
```

Comment:
Let's make this change separately
```suggestion
        rpcUrl: "https://volta-rpc.energyweb.org/",
```